packages/core-data/src/entities.js

```diff
@@ -35,6 +35,7 @@ function* loadPostTypeEntities() {
 			kind: 'postType',
 			baseURL: '/wp/v2/' + postType.rest_base,
 			name,
+			transientEdits: { _blocks: true },
```

Comment:
Why do we use `_` ? To denote that it's transient? Isn't it redundant?

Reply:
Separate from this PR: we should document the entity config somewhere

Reply:
Also, I know that some properties of the post object are not handled/merged separately (edits and persisted) and they're merged with a deep merge. Thinking of `meta` for instance (see the editor selectors..), I was thinking we probably need something here to "mark" these properties?

Reply:
>Why do we use _ ? To denote that it's transient? Isn't it redundant?

Yeah, it was in my head from the other PR, I'll remove it.

>Also, I know that some properties of the post object are not handled/merged separately (edits and persisted) and they're merged with a deep merge. Thinking of meta for instance (see the editor selectors..), I was thinking we probably need something here to "mark" these properties?

What if we just deeply merge edits if they are objects?

Reply:
> What if we just deeply merge edits if they are objects?

We discussed this a long time for the PR that introduced the behavior and I think there's a valid reason (object values that we don't want to merge recursively). I'll have to look for the original PR to find the details

Reply:
Related discussions here https://github.com/WordPress/gutenberg/pull/10827

Reply:
Mmmm, there will also be properties that should be merged to different depths. I think meta only merges shallowly.

What do you think of an `editsMergeDepths` object, which for now would have `meta: 1`.

Reply:
This could work yes.